zgw/documenten-api/src/main/kotlin/com/ritense/portal/documentenapi/domain/Document.kt

```diff
@@ -30,14 +29,14 @@ data class Document(
     val bestandsomvang: Int?,
     @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
     @GraphQLIgnore
-    val status: DocumentStatus?
+    val status: DocumentStatus?,
+    var documentapi: String = ""
```

Comment:
I like the fact that you're using a name or key as the identifier for the instance of the api. But you still expose the full domain of the documentenapi because you removed the `@GraphQLIgnore` from the `url` property. Is this on purpose?

Reply:
fixed